src/tailwind.css

```diff
@@ -1,8 +1,186 @@
 /* purgecss start ignore */
+
 @tailwind base;
 @tailwind components;
 /* purgecss end ignore */
 
 /* Override utils here*/
 
 @tailwind utilities;
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  background: #1d2530;
+}
+
+.joinUsWrapper {
+  display: flex;
+  flex-direction: column;
+}
+
+.joinUsWrapper__registerLoginWrapper {
+  display: flex;
+}
+
+.joinUsWrapper__registerLoginWrapper__register {
+  @apply -mx-5;
+  background: linear-gradient(270deg, #ffb421 11.7%, #ff7521 90.81%);
```

Comment:
since this gradient is prominent in your app, add it to a variable, and apply it where necessary. put inside a class called `.main-gradient`